test/functional/cmdLineTests/shareClassTests/SCCMLTests/ShareClassesCMLTests-6.xml

```diff
@@ -48,6 +48,13 @@
  	
 	<variable name="NON_WINDOWS_PLATFORMS" value="aix.*,linux.*,zos.*,osx.*" />
 	<variable name="WINDOWS_PLATFORMS" value="win.*" />
+	
+	<variable name="EXTRA_OPT_FOR_MPROTECT_NONE" value=" " />
+	<!-- memory protection is across processes on zos_390-31. If the cache is created not with -Xshareclasses:mprotect=none,
+		 it cannot be reused with -Xshareclasses:mprotect=none. Any update to it may leads to segmentation fault. Disable update
+		 using -Xaot:disableSharedCacheHints.
+	--> 
+	<variable name="EXTRA_OPT_FOR_MPROTECT_NONE" value="-Xaot:disableSharedCacheHints" platforms="zos_390-31.*" />
```

Comment:
I think we need this for both 31 and 64 bit.

Reply:
Those tests are already excluded on z/OS 64 bit:

https://github.com/eclipse-openj9/openj9/blob/5d74198dd822249f1fac4cb69deab74ee078a72b/test/functional/cmdLineTests/shareClassTests/SCCMLTests/exclude.xml#L123-L128